index.rst

```diff
@@ -0,0 +1,15 @@
+.. PPMS-PDFD documentation master file, created by
+   sphinx-quickstart on Tue Dec 12 14:22:39 2017.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Piwik PRO Developers documentation!
+=====================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Mobile SDK documentation
+   :name: test
+   :glob:
+
+   sdk/*
```

Comment:
I suggested to not use global linking. I would like to specify whole structure explicite to avoid situations where files are in unexpected sequence. This is also my strategy to hide deeper pages using `maxdepth` option.
Additionally `sdk` contains `md` files, but `config.py` expects `rst`. Does it work?

Reply:
This was suggested by RTD tutorial.